.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# All files should be reviewed by a member of the SDKs team
+* @stripe/api-library-reviewers
```

Comment:
There seems to be an error here. Do we have this group publicly accessible?

Reply:
yep 😁 in the PR description:

> This PR is blocked while the team is private

I'm figuring out who has the permissions

Reply:
ok, so it turns out teams are _never_ publicly accessible if you're not part of the team. But, the issue was the team didn't have "write" access (through the reviewers group directly; we got it through other teams we're all on). I've updated those permissions and it looks like we're good now!